test/controllers/addressLookup/AuthorisedOfficialsAddressLookupControllerSpec.scala

```diff
@@ -63,7 +63,7 @@ class AuthorisedOfficialsAddressLookupControllerSpec extends SpecBase with Befor
     mockAddressLookupConnector, inject[ErrorHandler], messagesControllerComponents)
 
   private val localUserAnswers: UserAnswers = emptyUserAnswers.set(
-    AuthorisedOfficialsNamePage(0), Name("Jim", Some("John"), "Jones")).success.value
+    AuthorisedOfficialsNamePage(0), Name(SelectTitle.values.head, "Jim", Some("John"), "Jones")).success.value
```

Comment:
SelectTitle.Mr would be more readable , please check all the places

Reply:
modified